epub33/epub-a11y-eaa-mapping/index.html

```diff
@@ -162,7 +161,11 @@ <h6>(i)</h6>
 								<p>Making the information available via more than one sensory channel</p>
 							</blockquote>
 
-							<p>Provision of information is covered by EPUB Accessibility, <a data-cite="epub-a11y-11#sec-discovery">section Discoverability</a>. Presentation of this information is out of scope of e-books, it is a requirement for reading systems, e-commerce websites, and platforms. </p>
+							<p>This requirement is satisfied by:</p>
+							<ul>
+								<li>WCAG Success Criterion <a href="https://www.w3.org/TR/WCAG2/#sensory-characteristics">1.3.3 Sensory Characteristics</a> (level A).</li>
+								<li>WCAG Success Criterion <a href="https://www.w3.org/TR/WCAG2/#use-of-color">1.4.1 Use of Color</a> (level A).</li>
+							</ul>
```

Comment:
I think many more WCAG criteria make "information available via more than one sensory channel". For example, text alternatives make visual information available as text, audio (via TTS), braille, etc. Same with captions and audio descriptions. Possibly also 1.3.1 Info and Relationships. I've not done a complete analysis, these are just a few that come to mind.

Reply:
The issue of alternative texts and audio-video are also present for the point "Supplementing any non-textual content with an alternative presentation of that content," do you think it is better to reassert?

Excellent insert 1.3.1 Info and Relationships.

Reply:
I didn't do a complete rundown but my feeling is that several WCAG Success Criteria relate to making information available in more than one sensory mode. If we start listing individual criteria then I think we should be comprehensive and list them all, even if it is repetitive.